met-api/src/met_api/models/survey.py

```diff
@@ -85,7 +85,7 @@ def create_survey(cls, survey: SurveySchema) -> DefaultMethodResult:
             created_by=survey.get('created_by', None),
             updated_by=survey.get('updated_by', None),
             engagement_id=survey.get('engagement_id', None),
-            is_hidden=survey.get('is_hidden', False),
+            is_hidden=survey.get('is_hidden', True),
```

Comment:
Lets keep default as false for now

Reply:
ok removed the logic and kept default as false